_extensions/gdscript.py

```diff
@@ -391,8 +333,1032 @@ def innerstring_rules(ttype):
                 ),
                 Name.Builtin.Type,
             ),
-        ],
-        "decorators": [
+            # copied from https://docs.godotengine.org/en/stable/classes/index.html
+            (
+                words(
+                    (
+                        # Nodes
+                        "Node",
+                        "AcceptDialog",
+                        "AnimatableBody2D",
+                        "AnimatableBody3D",
```

Comment:
I have concerns about this. It feels like this will grow out of date quickly and would need constant updates. If there is any way at all to automate this part, I'd appreciate it.

Reply:
While I agree automation would be nice (and I'll have a look into it), realistically I don't think this going out of date is much of an issue because 

1. The most important classes have had the same name since 4.x, they aren't renamed particularly often. 
2. New classes are rarely added to the docs right away
3. Classes are rarely removed between major updates too
4. If there are any outdated classes, the hue will only be off slightly
5. And considering all of that together, the benefit of covering most of the important classes far outweighs  risking missing a few less important ones

Reply:
Those are some fair points. :) Maybe we can make grabbing the list from that URL during the build a future todo then, and see how it goes.